storage/local/chunk/chunk.go

```diff
@@ -402,23 +409,23 @@ func newIndexAccessingChunkIterator(len int, acc indexAccessor) *indexAccessingC
 	return &indexAccessingChunkIterator{
 		len:       len,
 		pos:       -1,
-		lastValue: ZeroSamplePair,
+		lastValue: model.SamplePair{Timestamp: model.Earliest},
```

Comment:
Yes, sure, ZeroSample the same thing.

Only that the reference to #2032 is wrong. (Self-ref... ;)